examples/healthchecks/README.md

```diff
@@ -61,7 +61,16 @@ After we have a Cloud Spanner database created, we'll need a few variables:
 * Instance name
 * Database name aka DisplayName
 
-Once in, please edit the file `hc/local_settings.py` and make the section `DATABASES` into the following:
+Once in, please edit the file `hc/local_settings.py` to:
+a) Add `django_spanner` as the first entry to `INSTALLED_APPS`
+```python
+INSTALLED_APPS = [
+    'django_spanner',  # Must be listed first.
```

Comment:
Missing three dots on next line. Perhaps it would also be helpful to mention that the rest of the apps have to be copied from hc/settings.py as local_settings.py doesn't contain the INSTALLED_APPS setting.

Reply:
Done, thank you!